lib/extensions/map.dart

```diff
@@ -73,14 +73,14 @@ extension MapExtensions on Map {
   /// If value is NULL or not [List] type return default value [defaultString]
   ///
   // ignore: always_specify_types
-  Map getMap(String key) {
+  Map<String, dynamic> getMap(String key) {
```

Comment:
Let this extentsion be genric and you can create a new one called getSDMap Where SD stands for String,dynamic